controllers/featureflagconfiguration_controller.go

```diff
@@ -162,3 +179,32 @@ func (r *FeatureFlagConfigurationReconciler) finishReconcile(err error, requeueI
 	r.Log.Info("Finished Reconciling " + crdName)
 	return ctrl.Result{Requeue: true, RequeueAfter: interval}, nil
 }
+
+func (r *FeatureFlagConfigurationReconciler) GetFfReference(ff *configv1alpha1.FeatureFlagConfiguration) v1.OwnerReference {
+	return v1.OwnerReference{
+		APIVersion: ff.APIVersion,
+		Kind:       ff.Kind,
+		Name:       ff.Name,
+		UID:        ff.UID,
+		Controller: r.trueVal(),
+	}
+}
+
+func (r *FeatureFlagConfigurationReconciler) CheckOwnerReference(ff *configv1alpha1.FeatureFlagConfiguration, cm corev1.ConfigMap) bool {
+	for _, cmOwner := range cm.OwnerReferences {
+		if cmOwner.UID == r.GetFfReference(ff).UID {
+			return true
+		}
+	}
+	return false
+}
+
+func (r *FeatureFlagConfigurationReconciler) falseVal() *bool {
```

Comment:
Can you please make a `pkg/utils/utils.go` for these repetitive boolean pointers?

Reply:
done